sdk/v2/core/events.go

```diff
@@ -324,6 +331,22 @@ func (e *eventsClient) Get(
 	)
 }
 
+func (e *eventsClient) Clone(
+	ctx context.Context,
+	id string,
+) (EventList, error) {
+	events := EventList{}
+	return events, e.ExecuteRequest(
+		ctx,
+		rm.OutboundRequest{
+			Method:      http.MethodPut,
+			Path:        fmt.Sprintf("v2/events/%s/clone", id),
```

Comment:
REST nit. Post to `.../clones` might be better. Given that an event could be cloned multiple times, and therefor _have_ multiple clones, "clones" is a collection of sorts. Adding something to a collection is, more conventionally, a post.